exercises/concept/magician-in-training/.docs/instructions.md

```diff
@@ -1,93 +1,94 @@
 # Instructions
 
-As a magician-to-be, Elyse needs to practice some basics. She has a stack of cards that she wants to manipulate.
+As a magician-to-be, Elyse needs to practice some basics.
+She has a stack of cards that she wants to manipulate.
```

Comment:
```suggestion
She has a stack of cards that she wants to manipulate. The stack of cards can be represented by an array of strings, because regular cards contains numbers ( 1 to 10) and Jack, Queen and King, which we can denote with "J", "Q" and "K". To make it more fun you can use emojis. For example: "♦️J"
```